src/content/pages/ecosystem/index.md

```diff
@@ -203,6 +175,11 @@ community:
     description: Ask questions and engage with our community.
     url: https://discord.gg/Threshold
     rightIcon: /images/external-arrow.svg
+  - leftIcon: /images/telegram-social.svg
+    title: Telegram
+    description: Ask questions and engage with our community.
```

Comment:
Is it worth saying, "Ask questions and engage with our community.  Beware of scammers - we won't DM you first."